### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by Wydrys**

### CODE REVIEW

File: src/main/java/hello/controller/HelloController.java [hunk 1]
```diff
@@ -23,6 +23,9 @@ public HelloController(HelloService helloService) {
     public String greeting(@RequestParam(name = "name", defaultValue = "John") String name, Model model) {
         Person person = helloService.getPayload(name);
         model.addAttribute("name", person.getName());
+
+        System.out.println("Hello world");
```
Comment (jdebiec) [inline src/main/java/hello/controller/HelloController.java:27]: po co to

**Comment by KarolMis** [comment]
ok

File: src/main/java/hello/controller/HelloController.java (hunk 1 above)
Comment (KarolMis) [inline src/main/java/hello/controller/HelloController.java:27]: ok

Comment (jdebiec) [reply src/main/java/hello/controller/HelloController.java:27]: komentarz

File: src/main/java/hello/controller/HelloController.java [hunk 2]
```diff
@@ -23,6 +23,9 @@ public HelloController(HelloService helloService) {
     public String greeting(@RequestParam(name = "name", defaultValue = "John") String name, Model model) {
```
Comment (jdebiec) [inline src/main/java/hello/controller/HelloController.java:23]: co to

File: src/main/java/hello/controller/HelloController.java (hunk 1 above)
Comment (Wydrys) [inline src/main/java/hello/controller/HelloController.java]: dupa

File: src/main/java/hello/controller/HelloController.java [hunk 3]
```diff
@@ -23,6 +23,9 @@ public HelloController(HelloService helloService) {
...
         Person person = helloService.getPayload(name);
         model.addAttribute("name", person.getName());
```
Comment (jdebiec) [inline src/main/java/hello/controller/HelloController.java:25]: cos tam

File: src/main/java/hello/controller/HelloController.java [hunk 4]
```diff
@@ -23,6 +23,9 @@ public HelloController(HelloService helloService) {
     public String greeting(@RequestParam(name = "name", defaultValue = "John") String name, Model model) {
         Person person = helloService.getPayload(name);
```
Comment (jdebiec) [inline src/main/java/hello/controller/HelloController.java:24]: fdhfhfgh

**Pull request closed without merging**